gocov-xml.go

```diff
@@ -60,15 +66,34 @@ type Method struct {
 	LineHits   int64   `xml:"line-hits,attr"`
 }
 
+// Line information
 type Line struct {
 	Number int   `xml:"number,attr"`
 	Hits   int64 `xml:"hits,attr"`
 }
 
 func main() {
+	// check the arguments.
+	bpath, err := os.Getwd()
+	if err != nil {
+		panic(err)
+	}
+	if len(os.Args) > 2 {
+		if os.Args[1] == "-b" {
```

Comment:
Is there a reason not to use [`flag` package](https://golang.org/pkg/flag/)?

Reply:
👍 
Had merged from different branch.
I've now updated to use flag package.